File: WindowsApplication1/WindowsApplication1/Zapato.vb
Public Class Zapato
    Private Marca As String
    Private Talla As Integer

    Property Pmarca As String
        Get
            Return Marca
        End Get
        Set(value As String)
            Marca = value
        End Set
    End Property

    Property Ptalla As String
        Get
            Return Talla
        End Get
        Set(value As String)
            Talla = value
        End Set
    End Property
End Class

File: HCSZapato/HCSZapato/Zapato.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HCSZapato
{
    public class Zapato
    {
        private string marca;
        private int talla;

        public void setMarca(string m)
        {
            marca = m;
        }
        public string getMarca()
        {
            return marca;
        }
        public void setTalla(int t)
        {
            talla = t;
        }
        public int getTalla()
        {
            return talla;
        }
    }
}
